### PR TITLE
Release DataKit 0.10.0

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: packages/datakit-bridge-github/datakit-bridge-github.0.10.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "uri" {>= "1.8.0"}
+  "hvsock" {>= "0.8.1"}
+  "hex" "nocrypto" "conduit"
+  "prometheus-app"
+  "protocol-9p" {>= "0.8.0"}
+  "datakit-client" {>= "0.10.0"}
+  "github-hooks" {>= "0.1.1"}
+  "github" {>= "2.1.0"}
+  "datakit" {test & >= "0.10.0"}
+]
```
Comment (yallop) [inline packages/datakit-bridge-github/datakit-bridge-github.0.10.0/opam]: [There](https://travis-ci.org/ocaml/opam-repository/jobs/226867823)'s a missing dependency of some sort:

```
#=== ERROR while installing datakit-bridge-github.0.10.0 ======================#
[...]
# Error: Findlib package "datakit-server" not found.
# -> required by "tests/common/jbuild (context default)"
# Hint: try: jbuilder external-lib-deps --missing -p datakit-bridge-github -j 4
```

**Comment by samoht** [comment]
I've pushed a possible fix

**Comment by samoht** [comment]
Conduit issues should be fixed by https://github.com/ocaml/opam-repository/pull/9065

**Comment by samoht** [comment]
datakit-github-bridge issue should be fixed by https://github.com/ocaml/opam-repository/pull/9067

**Comment by samoht** [comment]
To speed up the feedback loop for fixing errors in old packages, I have opened https://github.com/ocaml/opam-repository/pull/9076 ...

**Comment by samoht** [comment]
Rebased

**Comment by samoht** [comment]
Remaining errors are infrastructure related, timeouts and solver weirdness:

```
[WARNING] External solver failed:
Cudf_parser.Parse_error("unexpected RFC 822 token", _)
[ERROR] External solver failure. This may be due to bad settings (solver or solver criteria) or a broken solver solver installation. Check $OPAMROOT/config, and the --external-solver and --solver-criteria options.
```

So merging.

**Pull request merged**